app/controllers/users_controller.rb

```diff
@@ -7,4 +7,20 @@ def show
   def new
     @user = User.new
   end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      redirect_to @user
+      flash[:success] = "Welcome to TrailBlazer!"
+    else
+      render 'new'
+    end
+  end
+
+  private
+
+    def user_params
```

Comment:
Most people don't indent after `private`, as it doesn't begin a block, and then it gets funny when it ends. Not a big deal, though.

Reply:
Fixed